baseframe/forms/auto.py

```diff
@@ -56,9 +57,12 @@ def render_form(
         if isinstance(field.widget, wtforms.widgets.FileInput):
             multipart = True
     if not with_chrome:
-        template = THEME_FILES[current_app.config['theme']]['ajaxform.html.jinja2']
+        if not layout_template:
+            layout_template = THEME_FILES[current_app.config['theme']][
+                'ajaxform.html.jinja2'
+            ]
```

Comment:
Wait, `ajaxform.html.jinja2` is not a layout template. It's a regular template. `formlayout.html.jinja2` is the layout template.